GUI/TaskWindows/SearchTaskWindow.xaml

```diff
@@ -440,7 +440,11 @@
                     <GroupBox Header="Output Options" DockPanel.Dock="Top" ScrollViewer.CanContentScroll="True">
                         <!--<Expander x:Name="OutputExpander">-->
                         <StackPanel Orientation="Vertical" Margin="5">
-                            <CheckBox x:Name="CkbMzId" Content="Write .mzID" />
+                            <CheckBox x:Name="CkbMzId" Content="Write .mzID" IsEnabled="{Binding IsChecked, ElementName=CheckBoxParsimony}" ToolTipService.ShowOnDisabled="True">
+                                <ToolTipService.ToolTip>
+                                    Output of .mzID requires protein parsimony. Please check "Apply protein parsimony and construct protein groups".
```

Comment:
Heads up that ToolTips don't display if the object is disabled, so the second part might not make sense (since they would only be able to read this if it was already checked).

Reply:
Line 443 ToolTipService.ShowOnDisabled="True" makes it show when disabled:
![image](https://user-images.githubusercontent.com/16841846/81066777-3ea47480-8ea3-11ea-94c4-fa5200ec9147.png)

Reply:
Nice! I was unfamiliar with that option. Thanks for pointing it out.